templates/web/merton/footer_extra.html

```diff
@@ -0,0 +1,11 @@
+<div class="societyworks-footer">
+    <div class="container site-footer">
+        <span>
+            Powered by
+            <a class="societyworks-footer__logo societyworks-footer__logo--fms-pro" href="https://www.societyworks.org/services/highways/">FixMyStreet Pro</a>
+        </span>
+        <span>
+            <a href="[%~ cobrand.privacy_policy_url ~%]">Privacy and Cookies</a> | <a href="[%~ cobrand.privacy_policy_url ~%]">Accessibility</a>
```

Comment:
The accessibility link isn't correct, it goes to the privacy policy again. It looked to be okay and then 07e8cbc1f5ca9863e2f234d18293461c22dad185 changed it for some reason?